index.html

```diff
@@ -3,8 +3,27 @@
   <head>
     <meta charset="utf-8">
     <title>js-calc</title>
+    <style>
+
+    </style>
   </head>
   <body>
+    <h1>Calculator</h1>
+    <div id="calc">
+      <p id="box"></p>
+      <p id="one">1</p>
+
+
+      <!-- <div>2</div>
+      <div>3</div>
+      <div>4</div>
+      <div>5</div>
+      <div>6</div>
+      <div>7</div> -->
+
+    <button onclick="addnumber()"> next</button>
+
+
     <script src="calculator.js" charset="utf-8"></script>
   </body>
 </html>
```

Comment:
If you want to redo this without doing stuff with the DOM before Friday, I'll accept it.